qualtran/_infra/bloq.py

```diff
@@ -466,6 +468,25 @@ def as_cirq_op(
             bloq=self, cirq_quregs=cirq_quregs, qubit_manager=qubit_manager
         )
 
+    def as_pl_op(self, wires: 'Wires') -> 'Operation':
+        """Override this method to support conversion to a PennyLane operation.
+
+        If this method is not overriden, the default implementation will wrap this bloq
+        in a `FromBloq` shim.
+
+        Args:
+            wires: the wires that the op acts on
+
+        Returns:
+            ~.Operation: A PennyLane operation corresponding to this bloq acting on the 
+                provided wires or None. This method should return None if and only if the bloq 
+                instance truly should not be included in the PennyLane circuit (e.g. for reshaping 
+                bloqs). A bloq with no PennyLane equivalent should raise an exception instead.
+        """
+        from pennylane.io import FromBloq
```

Comment:
Actually, is this circular dependancy going to be a problem while trying to merge these two PRs across two packages? To merge this current, Qualtran PR; we need to depend on a release of Pennylane that includes this adapter. But that adapter needs a release of qualtran with `Bloq.as_pl_op`. Am I capturing this correctly?